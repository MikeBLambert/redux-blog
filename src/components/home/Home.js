import React, { Component } from 'react';
import { fetchPostsPromise } from '../../actions/posts';

export default class Home extends Component {



  // componentDidMount() {
  //   fetchPostsPromise();
  // }

  render() {
    return (
      <div>
        <h1>Hello!</h1>
      </div>
    );
  }
}
