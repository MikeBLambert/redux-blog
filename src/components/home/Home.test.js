import Home from './Home';
import { shallow } from 'enzyme';
import React from 'react';

describe('Home component', () => {
  it('matches snapshot', () => {
    const wrapper = shallow(<Home />);
    expect(wrapper).toMatchSnapshot();
  });

});
