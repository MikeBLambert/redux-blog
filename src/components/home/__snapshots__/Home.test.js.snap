// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Home component matches snapshot 1`] = `
<div>
  <ul>
    <li
      key="0"
    >
      sunt aut facere repellat provident occaecatiexcepturi optio reprehenderit
    </li>
    <li
      key="1"
    >
      qui est esse
    </li>
    <li
      key="2"
    >
      ea molestias quasi exercitationem repellat qui ipsa sit aut
    </li>
    <li
      key="3"
    >
      eum et est occaecati
    </li>
    <li
      key="4"
    >
      nesciunt quas odio
    </li>
    <li
      key="5"
    >
      dolorem eum magni eos aperiam quia
    </li>
    <li
      key="6"
    >
      magnam facilis autem
    </li>
    <li
      key="7"
    >
      dolorem dolore est ipsam
    </li>
    <li
      key="8"
    >
      nesciunt iure omnis dolorem tempora et accusantium
    </li>
    <li
      key="9"
    >
      optio molestias id quia eum
    </li>
    <li
      key="10"
    >
      et ea vero quia laudantium autem
    </li>
    <li
      key="11"
    >
      in quibusdam tempore odit est dolorem
    </li>
    <li
      key="12"
    >
      dolorum ut in voluptas mollitia et saepe quoanimi
    </li>
    <li
      key="13"
    >
      voluptatem eligendi optio
    </li>
    <li
      key="14"
    >
      eveniet quod temporibus
    </li>
    <li
      key="15"
    >
      sint suscipit perspiciatis velit dolorum rerum ipsa laboriosam odio
    </li>
    <li
      key="16"
    >
      fugit voluptas sed molestias voluptatem provident
    </li>
    <li
      key="17"
    >
      voluptate et itaque vero tempora molestiae
    </li>
    <li
      key="18"
    >
      adipisci placeat illum aut reiciendis qui
    </li>
    <li
      key="19"
    >
      doloribus ad provident suscipit at
    </li>
    <li
      key="20"
    >
      asperiores ea ipsam voluptatibus modi minimaquia sint
    </li>
    <li
      key="21"
    >
      dolor sint quo a velit explicabo quia nam
    </li>
    <li
      key="22"
    >
      maxime id vitae nihil numquam
    </li>
    <li
      key="23"
    >
      autem hic labore sunt dolores incidunt
    </li>
    <li
      key="24"
    >
      rem alias distinctio quo quis
    </li>
    <li
      key="25"
    >
      est et quae odit qui non
    </li>
    <li
      key="26"
    >
      quasi id et eos tenetur aut quo autem
    </li>
    <li
      key="27"
    >
      delectus ullam et corporis nulla voluptas sequi
    </li>
    <li
      key="28"
    >
      iusto eius quod necessitatibus culpa ea
    </li>
    <li
      key="29"
    >
      a quo magni similique perferendis
    </li>
    <li
      key="30"
    >
      ullam ut quidem id aut vel consequuntur
    </li>
    <li
      key="31"
    >
      doloremque illum aliquid sunt
    </li>
    <li
      key="32"
    >
      qui explicabo molestiae dolorem
    </li>
    <li
      key="33"
    >
      magnam ut rerum iure
    </li>
    <li
      key="34"
    >
      id nihil consequatur molestias animi provident
    </li>
    <li
      key="35"
    >
      fuga nam accusamus voluptas reiciendis itaque
    </li>
    <li
      key="36"
    >
      provident vel ut sit ratione est
    </li>
    <li
      key="37"
    >
      explicabo et eos deleniti nostrum ab id repellendus
    </li>
    <li
      key="38"
    >
      eos dolorem iste accusantium est eaque quam
    </li>
    <li
      key="39"
    >
      enim quo cumque
    </li>
    <li
      key="40"
    >
      non est facere
    </li>
    <li
      key="41"
    >
      commodi ullam sint et excepturi error explicabo praesentium voluptas
    </li>
    <li
      key="42"
    >
      eligendi iste nostrum consequuntur adipisci praesentium sit beatae perferendis
    </li>
    <li
      key="43"
    >
      optio dolor molestias sit
    </li>
    <li
      key="44"
    >
      ut numquam possimus omnis eius suscipit laudantium iure
    </li>
    <li
      key="45"
    >
      aut quo modi neque nostrum ducimus
    </li>
    <li
      key="46"
    >
      quibusdam cumque rem aut deserunt
    </li>
    <li
      key="47"
    >
      ut voluptatem illum ea doloribus itaque eos
    </li>
    <li
      key="48"
    >
      laborum non sunt aut ut assumenda perspiciatis voluptas
    </li>
    <li
      key="49"
    >
      repellendus qui recusandae incidunt voluptates tenetur qui omnis exercitationem
    </li>
    <li
      key="50"
    >
      soluta aliquam aperiam consequatur illo quisvoluptas
    </li>
    <li
      key="51"
    >
      qui enim et consequuntur quia animi quis voluptate quibusdam
    </li>
    <li
      key="52"
    >
      ut quo aut ducimus alias
    </li>
    <li
      key="53"
    >
      sit asperiores ipsam eveniet odio non quia
    </li>
    <li
      key="54"
    >
      sit vel voluptatem et non libero
    </li>
    <li
      key="55"
    >
      qui et at rerum necessitatibus
    </li>
    <li
      key="56"
    >
      sed ab est est
    </li>
    <li
      key="57"
    >
      voluptatum itaque dolores nisi et quasi
    </li>
    <li
      key="58"
    >
      qui commodi dolor at maiores et quis id accusantium
    </li>
    <li
      key="59"
    >
      consequatur placeat omnis quisquam quia reprehenderit fugit veritatis facere
    </li>
    <li
      key="60"
    >
      voluptatem doloribus consectetur est ut ducimus
    </li>
    <li
      key="61"
    >
      beatae enim quia vel
    </li>
    <li
      key="62"
    >
      voluptas blanditiis repellendus animi ducimus error sapiente et suscipit
    </li>
    <li
      key="63"
    >
      et fugit quas eum in in aperiam quod
    </li>
    <li
      key="64"
    >
      consequatur id enim sunt et et
    </li>
    <li
      key="65"
    >
      repudiandae ea animi iusto
    </li>
    <li
      key="66"
    >
      aliquid eos sed fuga est maxime repellendus
    </li>
    <li
      key="67"
    >
      odio quis facere architecto reiciendis optio
    </li>
    <li
      key="68"
    >
      fugiat quod pariatur odit minima
    </li>
    <li
      key="69"
    >
      voluptatem laborum magni
    </li>
    <li
      key="70"
    >
      et iusto veniam et illum aut fuga
    </li>
    <li
      key="71"
    >
      sint hic doloribus consequatur eos non id
    </li>
    <li
      key="72"
    >
      consequuntur deleniti eos quia temporibus abaliquid at
    </li>
    <li
      key="73"
    >
      enim unde ratione doloribus quas enim ut sitsapiente
    </li>
    <li
      key="74"
    >
      dignissimos eum dolor ut enim et delectus in
    </li>
    <li
      key="75"
    >
      doloremque officiis ad et non perferendis
    </li>
    <li
      key="76"
    >
      necessitatibus quasi exercitationem odio
    </li>
    <li
      key="77"
    >
      quam voluptatibus rerum veritatis
    </li>
    <li
      key="78"
    >
      pariatur consequatur quia magnam autem omnisnon amet
    </li>
    <li
      key="79"
    >
      labore in ex et explicabo corporis aut quas
    </li>
    <li
      key="80"
    >
      tempora rem veritatis voluptas quo dolores vero
    </li>
    <li
      key="81"
    >
      laudantium voluptate suscipit sunt enim enim
    </li>
    <li
      key="82"
    >
      odit et voluptates doloribus alias odio et
    </li>
    <li
      key="83"
    >
      optio ipsam molestias necessitatibus occaecati facilis veritatis dolores aut
    </li>
    <li
      key="84"
    >
      dolore veritatis porro provident adipisci blanditiis et sunt
    </li>
    <li
      key="85"
    >
      placeat quia et porro iste
    </li>
    <li
      key="86"
    >
      nostrum quis quasi placeat
    </li>
    <li
      key="87"
    >
      sapiente omnis fugit eos
    </li>
    <li
      key="88"
    >
      sint soluta et vel magnam aut ut sed qui
    </li>
    <li
      key="89"
    >
      ad iusto omnis odit dolor voluptatibus
    </li>
    <li
      key="90"
    >
      aut amet sed
    </li>
    <li
      key="91"
    >
      ratione ex tenetur perferendis
    </li>
    <li
      key="92"
    >
      beatae soluta recusandae
    </li>
    <li
      key="93"
    >
      qui qui voluptates illo iste minima
    </li>
    <li
      key="94"
    >
      id minus libero illum nam ad officiis
    </li>
    <li
      key="95"
    >
      quaerat velit veniam amet cupiditate aut numquam ut sequi
    </li>
    <li
      key="96"
    >
      quas fugiat ut perspiciatis vero provident
    </li>
    <li
      key="97"
    >
      laboriosam dolor voluptates
    </li>
    <li
      key="98"
    >
      temporibus sit alias delectus eligendi possimus magni
    </li>
    <li
      key="99"
    >
      at nam consequatur ea labore ea harum
    </li>
  </ul>
</div>
`;
